blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -12,37 +12,40 @@ import * as pushEventUtil from './util/push-event-util';
 const { object, array, func, node, string } = PropTypes;
 
 class OrganizationPipelines extends Component {
-
-    // FIXME: IMO the following should be dropped
+    constructor(props) {
+        super(props);
+        this.state = {};
```

Comment:
state is evil, rule of thumb do not use it. ;) 

seriously, state is tricky since you cannot persist state of a component. Props you can persist!